wger/manager/templates/set/formset.html

```diff
@@ -1,5 +1,9 @@
 {% load wger_extras %}
 {% load i18n %}
+{% load staticfiles %}
+
```

Comment:
Why do you have blank spaces here
Kindly, give us a reason for this.

Reply:
I have made the adjustments. I believe it is okay now